src/clr-angular/forms/radio/radio-container.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {Component} from "@angular/core";
+
+import {DynamicWrapper} from "../../utils/host-wrapping/dynamic-wrapper";
+import {FormControlService} from "../common/form-control.service";
+
+@Component({
+    selector: "clr-radio-container",
+    template: `
+        <!-- We want the radio input to be before the label, always -->
+        <ng-content select="[clrRadio]"></ng-content>
+        <ng-content></ng-content>
+        <label *ngIf="_dynamic"></label>
+    `,
+    host: {"[class.radio]": "true"},
+    providers: [FormControlService]
+})
+export class ClrRadioContainer implements DynamicWrapper {
+    // Indicates whether the container is dynamically created by the radio button itself
+    _dynamic = false;
```

Comment:
Same question as above.
Is this supposed to be private, per our [guidlines](https://github.com/vmware/clarity/wiki/Naming-Conventions#public-vs-private) ?

Reply:
No, it is used in the template and therefore is `public`.